dev_tools/bash_scripts_test.py

```diff
@@ -59,7 +59,8 @@ def run(
     cmd = r"""
 dir=$(git rev-parse --show-toplevel)
 cd {}
-git init --quiet --initial-branch master
+mkdir empty-templates
+git init --template=empty-templates --quiet --initial-branch master
```

Comment:
Can you try if adding
```
export GIT_CONFIG_GLOBAL=/dev/null
export GIT_CONFIG_SYSTEM=/dev/null
```
at the start of the `cmd` would fix the issue as well?

This should make git run with its defaults, regardless of system and user configuration files.

Reply:
Tested this, and you're right, it works too. This is a better solution because it will prevent other possible user settings from interfering, not just the one for template dir. I'll edit the PR.